### PR TITLE
kubebuilder v2.0.0-alpha.1

### PR DESCRIPTION
**Pull request by stevemcquaid**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)? Yes
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change? Yes 
Previous PR I created: https://github.com/Homebrew/homebrew-cask/pull/63639
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting? Yes
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting? Yes
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)? Yes

-----

### CODE REVIEW

File: Formula/kubebuilder.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class Kubebuilder < Formula
+  desc "SDK for building Kubernetes APIs using CRDs"
+  homepage "https://github.com/kubernetes-sigs/kubebuilder"
+  url "https://github.com/kubernetes-sigs/kubebuilder.git",
+      :tag      => "v2.0.0-alpha.1",
```
Comment (SMillerDev) [inline Formula/kubebuilder.rb]: This doesn't seem to match: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Comment (stevemcquaid) [reply Formula/kubebuilder.rb]: The latest stable release uses a different package manager as well as a different install mechanism. Is there a tag that can be added to indicate its a alpha release?

Comment (zbeekman) [reply Formula/kubebuilder.rb]: No. My vote is we wait for upstream to mint a proper release. The other option is keep this in your back pocket for when they do, and submit the older release, using the older installation, although I'd rather not go that route.

You can also start your own tap for now, then re-submit to core once upstream has a stable release.

Comment (DirectXMan12) [reply Formula/kubebuilder.rb]: We'll be releasing a stable release soon (the plan was to let it soak for a week or two after KubeCon), so you shouldn't have to wait long :-)

**Comment by fxcoudert** [comment]
Closing for now, we'll be happy to accept a new PR as the stable version is released!

**Comment by alexkreidler** [comment]
excited for this!

**Comment by halfcrazy** [comment]
kuberbuilder release v2.0.0 https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v2.0.0

**Comment by stevemcquaid** [comment]
Looks like @cblecker already got this in: https://github.com/Homebrew/homebrew-core/commit/92e6941e3d51e501977995f7ae2c835cd647fc97

**Pull request closed without merging**